include/vkd3d_dxgiformat.idl

```diff
@@ -139,6 +139,63 @@ typedef enum DXGI_FORMAT
     DXGI_FORMAT_V208                        = 0x83,
     DXGI_FORMAT_V408                        = 0x84,
 
+    /* https://gli.g-truc.net/0.6.1/api/a00001.html */
+    DXGI_FORMAT_ASTC_4X4_TYPELESS           = 0x85,
```

Comment:
Given these formats are not documented, we shouldn't add them there. I'd suggest maybe adding a dummy DXGI_FORMAT_UNDOCUMENTED_ASTC_FIRST_FORMAT and LAST_FORMAT perhaps, and just range check it away.

Reply:
Fixed.